lib/plugins/login/login.js

```diff
@@ -47,115 +65,135 @@ class Login {
+                'RSA-OAEP'
+              );
+              const iv = keypair.privateKey.decrypt(
+                forge.util.decode64(response.cliLoginById.encryptedIv),
+                'RSA-OAEP'
+              );
+
+              const idToken = decryptToken(response.cliLoginById.encryptedIdToken, key, iv);
+              const accessToken = decryptToken(response.cliLoginById.encryptedAccessToken, key, iv);
+              const refreshToken = decryptToken(
+                response.cliLoginById.encryptedRefreshToken,
+                key,
+                iv
+              );
+
+              const decoded = jwtDecode(idToken);
+              this.serverless.cli.log('You are now logged in');
+              // because platform only support github
+              const id = decoded.tracking_id || decoded.sub;
+              const userConfig = {
+                userId: id,
+                frameworkId,
+                users: {},
+              };
+              // set user auth in global .serverlessrc file
+              userConfig.users[id] = {
+                userId: id,
+                name: decoded.name,
                 email: decoded.email,
-                force: true,
-              }).then(() => {
-                // then exit process
-                process.exit(0);
-              });
-            });
+                auth: {
+                  id_token: idToken,
+                  access_token: accessToken,
+                  refresh_token: refreshToken,
+                },
+              };
+
+              // update .serverlessrc
+              setConfig(userConfig);
+
+              // identify user for better onboarding
+              userStats
+                .identify({
+                  id,
+                  frameworkId,
+                  email: decoded.email,
+                  // unix timestamp
+                  created_at: Math.round(+new Date(decoded.createdAt) / 1000),
+                  trackingDisabled: configuration.trackingDisabled,
+                  force: true,
+                })
+                .then(() => {
+                  userStats
+                    .track('user_loggedIn', {
+                      id,
+                      email: decoded.email,
+                      force: true,
+                    })
+                    .then(() => {
+                      // then exit process
+                      process.exit(0);
+                    });
+                });
+            }
           })
           .catch(() => {
             this.serverless.cli.consoleLog(
-              chalk.red('Incorrect token value supplied. Please run "serverless login" again'));
-            process.exit(0);
+              chalk.red('Sorry, something went. Please run "serverless login" again.')
+            );
+            throw new this.serverless.classes.Error(
+              'Failed to login due an error. Please try again or contact hello@serverless.com'
```

Comment:
Just a quick question. Do we pipe everything through "hello@serverless.com" or are there any plans to add smth. like "support@serverless.com"?